src/components/apps/launch/AppLaunchForm.js

```diff
@@ -88,60 +100,56 @@ const StepperBottomNavigation = ({
-            value="back"
-            icon={<ArrowBack />}
-        />
-        {showSaveQuickLaunchButton ? (
-            <BottomNavigationAction
-                id={buildDebugId(formId, ids.BUTTONS.SAVE_AS_QUICK_LAUNCH)}
-                label={getMessage("saveAsQuickLaunch")}
-                value="saveQL"
-                icon={<Save />}
-            />
-        ) : (
-            <BottomNavigationAction disabled={true} />
-        )}
-        {showSubmitButton ? (
-            <BottomNavigationAction
-                id={buildDebugId(formId, ids.BUTTONS.SUBMIT)}
-                label={getMessage("launchAnalysis")}
-                value="submit"
-                icon={<PlayArrow />}
-            />
-        ) : (
-            <BottomNavigationAction
-                id={buildDebugId(formId, ids.BUTTONS.STEP_NEXT)}
-                label={getMessage("next")}
-                value="next"
-                icon={<ArrowForward />}
-            />
-        )}
-    </BottomNavigation>
-);
+}) => {
+    const classes = useStyles();
+    const theme = useTheme();
+    const isMobile = useMediaQuery(theme.breakpoints.down("xs"));
+    return (
+        <Toolbar variant="dense" className={classes.bottomNavigation}>
+            <Button
+                id={buildDebugId(formId, ids.BUTTONS.STEP_BACK)}
+                className={classes.bottomNavigationAction}
+                startIcon={<ArrowBack />}
+                size="small"
+                onClick={handleBack}
+            >
+                {isMobile ? "" : getMessage("back")}
+            </Button>
+            {showSaveQuickLaunchButton && (
+                <Button
+                    className={classes.bottomNavigationAction}
+                    id={buildDebugId(formId, ids.BUTTONS.SAVE_AS_QUICK_LAUNCH)}
+                    startIcon={<Save />}
+                    size="small"
+                    onClick={handleSaveQuickLaunch}
+                >
+                    {isMobile ? "" : getMessage("saveAsQuickLaunch")}
+                </Button>
+            )}
+            {showSubmitButton ? (
+                <Button
+                    className={classes.bottomNavigationAction}
+                    id={buildDebugId(formId, ids.BUTTONS.SUBMIT)}
+                    startIcon={<PlayArrow />}
+                    size="small"
+                    onClick={(event) => handleSubmit(event)}
+                >
+                    {isMobile ? "" : getMessage("launchAnalysis")}
+                </Button>
+            ) : (
+                <Button
+                    className={classes.bottomNavigationAction}
+                    id={buildDebugId(formId, ids.BUTTONS.STEP_NEXT)}
+                    endIcon={<ArrowForward />}
+                    size="small"
+                    onClick={handleNext}
+                >
+                    {isMobile ? "" : getMessage("next")}
+                </Button>
+            )}
+        </Toolbar>
+    );
+};
```

Comment:
I replaced `BottomNavigation` with `Toolbar` for few reasons:
1. `BottomNavigation` was not allowing me set `dense` padding 
2.  Removing labels caused icons to misalign
3.  I wasn't able to set Flex button layout.
4. Other minor `css` annoyances (that may be due to my css ignorance)